<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>545</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>930</width>
    <height>545</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>930</width>
    <height>545</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>930</width>
    <height>545</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tbwSeccion">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>921</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tabla</string>
     </attribute>
     <widget class="QTableWidget" name="twTable">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>621</width>
        <height>381</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;click en el titulo para ordenarlo&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string># Cuenta</string>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nombre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Apellido</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nota</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nota 2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nota 3</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nota 4</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nota 5</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Promedio</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="lblNumeroCuenta">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string># De Cuenta:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb1">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Determinar promedio&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblNota3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Nota 3:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leNota5">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Determinar promedio&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="bSaveData">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>410</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Guarda todo el contenido de la tabla&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
      <property name="icon">
       <iconset resource="Iconos.qrc">
        <normaloff>:/new/icons/Resources/save</normaloff>:/new/icons/Resources/save</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="lblNota2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nota 2:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leNota2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>190</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Determinar promedio&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblNota5">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>210</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Nota 5: </string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb5">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>230</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Determinar promedio&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="leNumeraCuenta">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>ingrese # cuenta</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leNota4">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>190</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="leNota1">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="leNota3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>140</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblNombre">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leNombre">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>ingrese nombre</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bAddRow">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>310</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Agrega los datos a la tabla&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
      <property name="icon">
       <iconset resource="Iconos.qrc">
        <normaloff>:/new/icons/Resources/add</normaloff>:/new/icons/Resources/add</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="lblNota1">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Nota 1:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblApellido">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Apellido:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>140</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Determinar promedio&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblPromedio">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>260</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Promedio: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leApellido">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>ingrese apellido</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblNota4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>170</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Nota 4:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lePromedio">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>280</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>ingrese notas</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bDeleteRow">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>410</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Elimina un elemento de la tabla&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
      <property name="icon">
       <iconset resource="Iconos.qrc">
        <normaloff>:/new/icons/Resources/delete</normaloff>:/new/icons/Resources/delete</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="bCambiar">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>410</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Elimina un elemento de la tabla&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Cambiar</string>
      </property>
      <property name="icon">
       <iconset resource="Iconos.qrc">
        <normaloff>:/new/icons/Resources/change.ico</normaloff>:/new/icons/Resources/change.ico</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Búsqueda</string>
     </attribute>
     <widget class="QTableWidget" name="twTable_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>901</width>
        <height>401</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string># Cuenta</string>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nombre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Apellido</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nota</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nota 2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nota 3</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nota 4</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nota 5</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Promedio</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="lebusqueda">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>430</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Escriba el nombre a buscar&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>430</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Búsqueda</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Iconos.qrc"/>
 </resources>
 <connections/>
</ui>
